OpenOversight/tests/routes/test_descriptions.py

```diff
@@ -13,36 +14,36 @@
 @pytest.mark.parametrize(
     "route",
     [
-        ("officer/1/description/1/edit"),
-        ("officer/1/description/new"),
-        ("officer/1/description/1/delete"),
+        "officer/1/description/1/edit",
+        "officer/1/description/new",
+        "officer/1/description/1/delete",
     ],
 )
 def test_route_login_required(route, client, mockdata):
     rv = client.get(route)
-    assert rv.status_code == 302
+    assert rv.status_code == HTTPStatus.FOUND
 
 
 @pytest.mark.parametrize(
     "route",
     [
-        ("officer/1/description/1/edit"),
-        ("officer/1/description/new"),
-        ("officer/1/description/1/delete"),
+        "officer/1/description/1/edit",
+        "officer/1/description/new",
+        "officer/1/description/1/delete",
```

Comment:
This sort of parenthesis is redundant, so I removed them.

Reply:
Good call. I was surprised to learn that black doesn't do that already, so I quickly looked into it. Turns out the reason is that black sometimes uses redundant parenthesis to break things up into multiple lines which makes this a little more complicated

Reply:
Ayyyye, it does that a lot when the lines are longer than whatever the line limit is.